values.yaml

```diff
@@ -9,6 +9,7 @@ OCP_PORT: ""
 OCP_SERVICENAME: ""
```

Comment:
Need to remove this as well @shantamkumolus

Reply:
OK :+1: